_includes/tutorials/connect-sink-timestamp/ksql/code/tutorial-steps/dev/select-from-stream.sql

```diff
@@ -0,0 +1,4 @@
+SELECT TEMPERATURE, CONVERT_TZ(FROM_UNIXTIME(EVENTTIME), 'UTC', 'America/Denver') AS EVENTTIME_MT
```

Comment:
Before showing it as Denver timezone, would it be helpful to first show the following (without timezone change) to drive home the point that these are 2 distinct steps/functions?

```
SELECT TEMPERATURE, FROM_UNIXTIME(EVENTTIME) AS
....
```

Reply:
I broke out this tutorial into more steps so that it's not one nested query. I did keep the nested query as the "short answer"